tools/system_libs.py

```diff
@@ -1069,7 +1077,8 @@ class Dummy(object):
   # You can provide 1 to include everything, or a comma-separated list with the ones you want
   force = os.environ.get('EMCC_FORCE_STDLIBS')
   if force == '1':
-    force = ','.join(system_libs_map.keys())
+    all_libs = system_libs_map.keys()
+    force = ','.join(l for l in all_libs if 'embind' not in l)
```

Comment:
I did something similar for #8711, but I think the solution is more general. I split it out into a new PR (#8821) so you can use it here as well.

Reply:
Done